shieldlib/src/shield.d.ts

```diff
@@ -15,7 +15,8 @@ export function storeNoShield(
 export function missingIconLoader(
   renderContext: ShieldRenderingContext,
   routeDef: RouteDefinition,
-  spriteID: string
+  spriteID: string,
+  update?: boolean
```

Comment:
Did you perhaps forget to commit shield.js

Reply:
What is missing?